python/samples/getting_started/00-getting-started.ipynb

```diff
@@ -20,6 +20,72 @@
     "%pip install semantic-kernel==1.1.0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Config\n",
+    "\n",
+    "Let's get started with the necessary configuration to run Semantic Kernel. For Notebooks, we require a `.env` file with the proper settings for the model you use. You may copy the `.env.example` file from this directory and place it either in this current directory, or in the main semantic-kernel/python directory.\n",
+    "\n",
+    "If interested, as you learn more about Semantic Kernel, there are a few other ways to make sure your secrets, keys, and settings are used:\n",
+    "\n",
```

Comment:
I think we should put this in a separate docs file with info on how to configure the kernel. That way we can also include information on using a token or token_provider of Azure, etc. The line above this can link to that file.

Reply:
Good idea, working on it now.